src/openzaak/utils/query.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models import QuerySet
+
+
+class BlockChangeMixin:
+    @property
+    def msg(self):
+        return f"This method is forbidden for {self.model.__name__} queryset"
+
+    def bulk_create(self, objs, batch_size=None, ignore_conflicts=False):
+        raise TypeError(self.msg)
```

Comment:
I'd probably change `TypeError` into a custom exception - it's a bit too generic and doesn't fit the purpose exactly.